sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -124,9 +183,9 @@
       </group>
       <rosparam unless="$(arg hand_ctrl)" command="load" file="$(find sr_ur_launch)/config/ra_trajectory_controller.yaml"/>
       <node if="$(arg hand_ctrl)" name="arm_trajectory_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
-            args="--wait-for=/$(arg arm_id)_arm_ready --wait-for=/calibrated $(arg arm_id)_trajectory_controller"/>
+            args="--wait-for=/$(arg arm_id)_sr_ur_robot_hw/robot_program_running $(arg arm_id)_trajectory_controller"/>
```

Comment:
This will actually be an issue and it can't be merged.
We might need to modify the spawner code so that it accepts 2 topics to `wait-for`.
Remember that passing the argument twice, as we were doing, actually doesn't work (the second argument overwrites the first).
We could for example modify the spawner to allow for a string of comma-separated topics.
It would still have to work with a single topic, of course.

Reply:
Is this still the case since we'll be modifying joint_state_publisher.launch?